osu.Game.Rulesets.Tau/Objects/Drawables/Pieces/HardBeatPiece.cs

```diff
@@ -1,21 +1,24 @@
-﻿using osu.Framework.Bindables;
+﻿using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Utils;
+using osu.Game.Rulesets.Tau.Configuration;
 using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Tau.Objects.Drawables.Pieces
 {
     public partial class HardBeatPiece : CircularContainer
     {
         public BindableFloat NoteSize = new(16f);
+        public BindableBool HighlightHardBeats = new(false);
 
         public HardBeatPiece()
         {
             Masking = true;
             BorderThickness = 5;
-            BorderColour = Color4.White;
+            BorderColour = HighlightHardBeats.Value ? Color4.Orange : Color4.White;
```

Comment:
This will always be false, and is redundant.

The correct colour will be assigned when the config is bound in load (via `BindValueChanged`)